.isort.cfg

```diff
@@ -0,0 +1,2 @@
+[settings]
+force_single_line=True
```

Comment:
Let's rather do,
> You can also add your desired settings under a [tool.isort] section in your pyproject.toml file.

To avoid new config files. We should add it to `.pre-commit-config.yaml` -- I can make a PR.